plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/sdk/AppEngineSDKClasspathContainer.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.sdk;
+
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.jdt.core.IClasspathContainer;
+import org.eclipse.jdt.core.IClasspathEntry;
+import org.eclipse.jdt.core.JavaCore;
+
+import com.google.cloud.tools.appengine.cloudsdk.PathResolver;
+
+public final class AppEngineSDKClasspathContainer implements IClasspathContainer {
+
+  private static final String SDK_JAR = "/platform/google_appengine/google/appengine/tools/java/lib/appengine-tools-api.jar";
```

Comment:
soon, if not yet, we can all exceot the last piece form app-tools-lib. Please add a todo.